src/components/TextField/TextField.tsx

```diff
@@ -26,23 +27,19 @@ export interface TextFieldProps extends BaseTextFieldProps, DebouncedOnChangePro
   underlined?: boolean;
 }
 
-export interface TextFieldComponent {
-  /**
-   * Sets focus on the input.
-   */
-  focus(): void;
-}
-
 /**
  * The TextField component enables a user to type text into an app. It's used to capture
  * a single line of text. The text displays on the screen in a simple, uniform format.
  */
-class TextField extends React.Component<TextFieldProps> implements TextFieldComponent {
+class TextField extends React.Component<TextFieldProps> implements Focusable {
   private fabricTextFieldRef: ITextField | null;
```

Comment:
could we also use the `createRef` API here?

Reply:
Unsure - createRef is designed for use with React refs, which get you a ref the the underlying React element. This thing is more of a Fabric convention that gets you a helper interface. It might work, but I figured I'd leave it alone since it's not the same mechanism.